### ISSUE TITLE
❓ What are the actual alpha2, alpha3, alpha4 values used in experiments?

### ISSUE THREAD

**Issue by Wendyywc** [DingjieFu/CVsC#2]
I'm trying to reproduce the results reported in your paper (Table 1), but I 
noticed that in the default config (`parameter.py`), the core loss weights 
are all set to 0:

```python
alpha2 = 0.0  # CCL
alpha3 = 0.0  # SCI  
alpha4 = 0.0  # TCI
```

**Could you please share the actual hyperparameter values you used to achieve 
the reported results?** Specifically:
- `alpha2` (CCL weight) = ?
- `alpha3` (SCI weight) = ?
- `alpha4` (TCI weight) = ?

And are there any other important hyperparameters (learning rate schedule, 
number of epochs, etc.) that differ from the defaults?

This would be very helpful for reproducing your results. Thank you!

**Comment by DingjieFu** [comment]
Hi, I'm sorry for the late reply.

For different datasets:
CUB: --alpha2 0.4 --alpha3 0.5 --alpha4 1.5
SUN: --alpha2 0.1 --alpha3 0.2 --alpha4 0.8
AWA2: --alpha2 0.4 --alpha3 0.2 --alpha4 1.5

Additionally, the value of alpha0, alpha1, and bias also affect the results. If you're trying to get the best performance, just use `wandb` to tune hyper-parameters.
